abs.yaml

```
@@ -0,0 +1,2 @@
+upload_channels:
+  - https://staging.continuum.io/prefect/fs/pyphen-feedstock/pr2/25bf479
```

when pyphen is done:

```suggestion
upload_channels:
  - sfe1ed40
```